README.md

```diff
@@ -239,30 +317,84 @@ can determine the _order_ in which they need to be worked on.
 
 ### C) _Estimate_ the Time Required to Perform the Work
 
-+ During "_Sprint Planning_" a ***time estimate label*** is added
-  to each story (_and Sub-Stories/tasks_)
-  The time estimate _must_ include time to:
-  + ***document***
-  + ***test***
-  + ***develop*** and
-  + ***deliver***
+1. For an _ongoing_ project, we estimate stories
+  During "_Sprint Planning_" session; <br />
+  A ***time estimate label*** (_e.g_: `T4h`  or `T25m`) is added
+  to each story (_and any Sub-Stories/tasks_) <br />
+  The time estimate **_must_ factor** time to:
+  + ***document*** the changes made to the codebase or UI
+  + ***test*** both the code (_unit/functional tests_) and UI tests
+  + ***develop*** the feature (_or fix the "bug"_)
+  + ***deliver*** by submitting a PR and going through a review/QA cycle.
+  + ***deploy*** the work to an environment
+  where a non-technical peopler can _evaluate_ it against the original
+  story description and acceptance criteria!
+
 + Where ever possible/practical/relevant **include** the person who
```

Comment:
indeed. my engrish isn't greight... please collect my typos when you do the next iteration. thanks.